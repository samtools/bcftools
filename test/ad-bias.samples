NA12890	NA12891
NA12890	NA12892
