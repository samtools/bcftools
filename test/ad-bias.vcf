##fileformat=VCFv4.2
##FILTER=<ID=PASS,Description="All filters passed">
##contig=<ID=22,length=51304566>
##FORMAT=<ID=AD,Number=R,Type=Integer,Description="Allelic depths">
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	NA12890	NA12891	NA12892
22	16050159	.	C	T	68	.	.	GT:AD	0/0:4,0	0/1:6,8	0/0:26,0
22	16050252	.	A	T	85	.	.	GT:AD	0/0:3,0	0/1:13,13	0/0:40,0
22	16051249	.	T	C	32.1761	.	.	GT:AD	0/1:4,3	0/0:41,0	0/0:55,0
22	16051347	.	G	C	283	.	.	GT:AD	1/1:2,4	0/1:20,20	0/0:43,0
22	16051453	.	A	C	3.42882	.	.	GT:AD	0/1:5,4	0/0:39,0	0/0:47,0
22	16051497	.	A	G	362	.	.	GT:AD	1/1:0,8	0/1:14,20	0/0:44,0
22	16051968	.	C	A	136	.	.	GT:AD	0/0:4,0	0/1:11,11	0/0:13,0
22	16052167	.	AAAACAAACAAACAAACAAACAAACAAACAAAC	AAAACAAACAAACAAACAAACAAACAAACAAACAAACAAAC	217	.	.	GT:AD	./.:0,0	1/1:0,27	0/0:46,0
22	16052169	.	AACAAACA	AACAAACAGACAAACA	241	.	.	GT:AD	0/1:1,1	0/1:7,29	0/0:54,0
22	16052180	.	AAAC	AAACCAACTAAC	221	.	.	GT:AD	0/0:2,0	0/1:16,15	0/0:55,0
22	16052239	.	A	G	172	.	.	GT:AD	0/0:5,0	0/1:12,17	0/0:44,0
22	16052513	.	G	C	153	.	.	GT:AD	0/0:11,0	0/1:25,16	0/0:38,0
22	16052618	.	G	A	124	.	.	GT:AD	0/0:7,0	0/1:29,12	0/0:41,0
22	16053659	.	A	C	352	.	.	GT:AD	1/1:0,9	1/1:0,21	0/0:30,0
22	16053791	.	C	A	287	.	.	GT:AD	0/1:4,4	0/1:13,16	0/0:32,0
22	16054454	.	C	T	14.2772	.	.	GT:AD	0/1:2,2	0/0:34,0	0/0:31,0
22	16054667	.	C	G	179	.	.	GT:AD	0/0:8,0	0/1:12,15	0/0:27,0
22	16054740	.	A	G	126	.	.	GT:AD	0/0:6,0	0/1:11,16	0/0:26,0
22	16055942	.	C	T	385	.	.	GT:AD	1/1:0,7	1/1:0,22	0/0:31,0
22	16056126	.	G	A	263	.	.	GT:AD	0/1:2,6	0/1:12,16	0/0:27,0
22	16056854	.	G	GA	17.3105	.	.	GT:AD	0/0:7,0	0/1:11,15	0/0:35,0
22	16057248	.	G	A	167	.	.	GT:AD	0/0:5,0	0/1:15,14	0/0:32,0
22	16057320	.	G	A	20.0168	.	.	GT:AD	0/1:2,2	0/0:25,0	0/0:36,0
22	16057417	.	C	T	16.3704	.	.	GT:AD	0/1:2,2	0/0:36,0	0/0:25,0
22	16058070	.	A	G	452	.	.	GT:AD	1/1:0,4	1/1:0,21	1/1:0,22
22	16058415	.	A	G	212	.	.	GT:AD	0/0:9,0	1/1:0,37	0/0:35,2
22	16058463	.	C	T	136	.	.	GT:AD	0/1:1,8	0/0:40,0	0/0:45,0
22	16058758	.	C	A	107	.	.	GT:AD	1/1:2,8	0/0:19,0	0/0:34,0
22	16058766	.	G	A	15.517	.	.	GT:AD	0/0:11,0	1/1:0,4	0/0:32,0
22	16058852	.	A	T	198	.	.	GT:AD	1/1:0,3	1/1:0,19	0/0:36,0
22	16058883	.	A	G	129	.	.	GT:AD	1/1:0,1	1/1:0,18	0/0:36,0
22	16059081	.	A	G	212	.	.	GT:AD	0/0:6,0	1/1:0,35	0/0:35,0
22	16059734	.	C	T	114	.	.	GT:AD	1/1:1,9	0/0:23,0	0/0:31,0
22	16059753	.	A	T	212	.	.	GT:AD	0/0:9,0	1/1:0,19	0/0:28,0
22	16059973	.	C	A	195	.	.	GT:AD	0/0:4,0	0/0:21,0	0/1:21,18
22	16060178	.	G	A	187	.	.	GT:AD	0/0:5,0	1/1:1,20	0/0:36,0
