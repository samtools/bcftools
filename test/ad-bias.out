#
# FT, Fisher Test	[2]Sample	[3]Control	[4]Chrom	[5]Pos	[6]smpl.nREF	[7]smpl.nALT	[8]ctrl.nREF	[9]ctrl.nALT	[10]P-value
FT	NA12890	NA12892	22	16051249	4	3	55	0	9.254363e-04
FT	NA12890	NA12892	22	16051347	2	4	43	0	7.079613e-05
FT	NA12890	NA12891	22	16051453	5	4	39	0	6.475486e-04
FT	NA12890	NA12892	22	16051453	5	4	47	0	3.430532e-04
FT	NA12890	NA12892	22	16051497	0	8	44	0	1.328836e-09
FT	NA12890	NA12892	22	16053659	0	9	30	0	4.718870e-09
FT	NA12890	NA12892	22	16053791	4	4	32	0	7.659481e-04
FT	NA12890	NA12892	22	16055942	0	7	31	0	7.923770e-08
FT	NA12890	NA12892	22	16056126	2	6	27	0	1.725030e-05
FT	NA12890	NA12891	22	16058415	9	0	0	37	9.076747e-10
FT	NA12890	NA12891	22	16058463	1	8	40	0	1.995662e-08
FT	NA12890	NA12892	22	16058463	1	8	45	0	8.649972e-09
FT	NA12890	NA12891	22	16058758	2	8	19	0	1.048427e-05
FT	NA12890	NA12892	22	16058758	2	8	34	0	2.539036e-07
FT	NA12890	NA12891	22	16058766	11	0	0	4	7.326007e-04
FT	NA12890	NA12892	22	16058852	0	3	36	0	1.094212e-04
FT	NA12890	NA12891	22	16059081	6	0	0	35	2.224007e-07
FT	NA12890	NA12891	22	16059734	1	9	23	0	2.592884e-07
FT	NA12890	NA12892	22	16059734	1	9	31	0	2.854341e-08
FT	NA12890	NA12891	22	16059753	9	0	0	19	1.447828e-07
FT	NA12890	NA12891	22	16060178	5	0	1	20	9.121313e-05
# SN, Summary Numbers	[2]Number of Pairs	[3]Number of Sites	[4]Number of comparisons	[5]P-value output threshold
SN	2	36	72	1.000000e-03
